packages/ros1/src/SystemHandle.cpp

```diff
@@ -154,13 +154,14 @@ bool SystemHandle::subscribe(
     const std::string& topic_name,
     const std::string& message_type,
     SubscriptionCallback callback,
-    const YAML::Node& /*configuration*/)
+    const YAML::Node& configuration)
 {
-  // TODO(MXG): Parse configuration so users can change queue size and transport
-  // hints
+  const YAML::Node& queue_config = configuration["queue_size"];
+  int queue_size = queue_config ? queue_config.as<int>() : default_queue_size;
```

Comment:
Super nitpick, but we can replace these two lines with just:

```suggestion
  int queue_size = configuration["queue_size"].as<int>(default_queue_size);
```